build-tools/installers/create-installers.targets

```diff
@@ -165,6 +165,7 @@
     <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)libZipSharp.dll" />
     <_MSBuildFiles Include="@(_LocalizationLanguages->'$(MicrosoftAndroidSdkOutDir)%(Identity)\libZipSharp.resources.dll')" />
     <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)libZipSharp.pdb" />
+    <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)MavenNet.dll" />
```

Comment:
As we're redistributing MavenNet, we'll need to update `THIRD-PARTY-NOTICES.txt` to mention it.  See also:

  * https://github.com/xamarin/xamarin-android/blob/2b0761b159f679608f1d9a5c5ec186121471cfc7/build-tools/xaprepare/xaprepare/Steps/Step_ThirdPartyNotices.cs#L79
  * https://github.com/xamarin/xamarin-android/tree/main/build-tools/xaprepare/xaprepare/ThirdPartyNotices
  * https://github.com/xamarin/xamarin-android/blob/2b0761b159f679608f1d9a5c5ec186121471cfc7/build-tools/xaprepare/xaprepare/ThirdPartyNotices/Java.Interop.cs